src/Symfony/Bundle/FrameworkBundle/Console/Application.php

```diff
@@ -70,12 +73,28 @@ public function doRun(InputInterface $input, OutputInterface $output)
 
         $this->setDispatcher($this->kernel->getContainer()->get('event_dispatcher'));
 
+        if ($this->registrationErrors) {
+            $this->renderRegistrationErrors($output);
```

Comment:
Of course :) thx.

Still.. this could also happen for lazy commands from the command loader. I dont think thats covered and happens in between currently...